contracts/teller-network/Escrow.sol

```diff
@@ -61,43 +70,113 @@ contract Escrow is Pausable, MessageSigned {
 
     enum ArbitrationResult {UNSOLVED, BUYER, SELLER}
 
+
     /**
      * @notice Create a new escrow
-     * @param _buyer The address that will perform the buy for the escrow
-     * @param _amount How much ether/tokens will be put in escrow
+     * @param _buyer Buyer address
+     * @param _seller Seller address
+     * @param _tradeAmount Amount buyer is willing to trade
+     * @param _tradeType Indicates if the amount is in crypto or fiat
      * @param _token Token address. Must be 0 for ETH
+     * @dev Requires contract to be unpaused.
+     *         The seller needs to be licensed.
+     */
+    function create(address payable _buyer, address payable _seller, address _token, uint _tradeAmount, uint8 _tradeType) public whenNotPaused {
+        require(msg.sender == _buyer || msg.sender == _seller, "Must participate in the trade");
+        require(license.isLicenseOwner(_seller), "Must be a valid seller to create escrow transactions");
```

Comment:
Should we also verify that the offer exists?

Reply:
We could, but it's an additional layer of complexity. Also, this way let's the seller negotiate and do whatever escrow he wants. 
In the bad case where the buyer "cheats" and puts a lower price, the seller can just not fund the escrow.

Reply:
fair enough